scripts/build/Platform/Android/pipeline.json

```diff
@@ -1,7 +1,7 @@
 {
     "ENV": {
         "GRADLE_HOME": "C:/Gradle/gradle-7.0",
-        "NODE_LABEL": "windows-a4a28adb2",
+        "NODE_LABEL": "windows-8fe4037c",
```

Comment:
Will we need to clear out the current snapshots after this is merged in?

Reply:
I'm creating a snapshot job to bump up the latest snapshot (https://jenkins.build.o3de.org/view/Tools/job/O3DE-development_snapshot/), but will rename the old snapshots for a short period as a fallback mechanism.